app/stac_api/models.py

```diff
@@ -289,15 +263,10 @@ class Meta:
     # using "name" instead of "id", as "id" has a default meaning in django
     name = models.CharField('id', unique=True, max_length=255, validators=[validate_name])
     created = models.DateTimeField(auto_now_add=True)
-    updated = models.DateTimeField(auto_now=True)
+    updated = models.DateTimeField(auto_now_add=True)
```

Comment:
maybe add a comment why you use `auto_now_add` here instead of `auto_now`

Reply:
done